.github/workflows/windows-build-and-test.yaml

```diff
@@ -65,10 +65,12 @@ jobs:
             pkg_version: ${{ fromJson(needs.config.outputs.pg13_latest) }}
             tsl_skips_version: dist_grant-13 dist_partial_agg-13
           - pg: 14
-            pkg_version: ${{ fromJson(needs.config.outputs.pg14_latest) }}
+            # pkg_version: ${{ fromJson(needs.config.outputs.pg14_latest) }}
+            pkg_version: 14.5.1 # hardcoded due to issues with PG14.6 on chocolatey
             tsl_skips_version: dist_partial_agg-14 dist_grant-14
           - pg: 15
-            pkg_version: ${{ fromJson(needs.config.outputs.pg15_latest) }}
+            # pkg_version: ${{ fromJson(needs.config.outputs.pg15_latest) }}
+            pkg_version: 15.0.1 # hardcoded due to issues with PG15.1 on chocolatey
```

Comment:
I'm not sure about these details myself, just trying to backport the fix Konstantina did for main branch into the 2.10.x branch, so that I can release 2.10.2 this week.